utils/deploy.ts

```diff
@@ -371,13 +394,16 @@ export const seedContracts = async ({
         hre,
         removalData: {
           supplierAddress: hre.namedAccounts.supplier,
+          vintage: 3000, // avoid clashing with likely test vintages and getting TokenIdExists error
```

Comment:
can you expand on this comment?

Reply:
yeah... just an attempt to avoid creating duplicate token ids, but I think I can remove this entirely and use the randomly generated subidentifer to keep token ids unique if necessary